5.validation_plate/0.illumination_correction/perform_ic.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
+conda init bash
+# activate the CellProfiler environment
+conda activate fibrosis_cp_env
+
+# convert Jupyter notebook(s) to script
+jupyter nbconvert --to script --output-dir=nbconverted/ *.ipynb
```

Shellcheck (0.11.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)